implement global variables + callbackUrl support (#246)
Mintbase/mintbase-js: packages/app/README.md

## Getting Started

Mintbase JS Browser/Next testing suite. 

# config vars

> You can set and NEED to set the Near Network you gonna use on your app in three ways:

Reviewer: ```suggestion
You may specify the NEAR network you want to interact with in three ways:
```
